src/libs/ReportActionsUtils.ts

```diff
@@ -1217,12 +1319,15 @@ function isApprovedOrSubmittedReportAction(action: OnyxEntry<ReportAction> | Emp
  * Gets the text version of the message in a report action
  */
 function getReportActionMessageText(reportAction: OnyxEntry<ReportAction> | EmptyObject): string {
+    if (!Array.isArray(reportAction?.message)) {
+        return '';
+    }
     return reportAction?.message?.reduce((acc, curr) => `${acc}${curr?.text}`, '') ?? '';
```

Comment:
Here I think we should get the `text` from `message` to ensure nothing is broken after phase 2.